lib/batcher.js

```diff
@@ -1,34 +1,34 @@
-
-
 function createBatcher(handleBatch, maxSize, maxTime) {
   return {
     dataArray: [],
     // using closure, so no need to keep as part of the object.
     // maxSize: maxSize,
     // maxTime: maxTime,
-    add: function(data) {
+    /** @type {any} */
+    add: function (data) {
       this.dataArray.push(data);
       if (this.dataArray.length >= maxSize) {
         this.flush();
       } else if (maxTime && this.dataArray.length === 1) {
         var self = this;
-        this._timeout = setTimeout(function() {
+        this._timeout = setTimeout(function () {
           self.flush();
         }, maxTime);
       }
     },
-    flush: function() {
+    /** @type {any} */
```

Comment:
This type hint is needed to avoid ```TS9005: Declaration emit for this file requires using private name 'flush'. An explicit type annotation may unblock declaration emit.```